.github/workflows/regressions.yml

```diff
@@ -0,0 +1,72 @@
+name: Regressions
+on: workflow_dispatch
+
+env:
+  PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 1
+  PUPPETEER_EXECUTABLE_PATH: /usr/bin/google-chrome
+  FORCE_COLOR: 1
+
+jobs:
+  build:
+    name: Build W3C profile
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

Comment:
so olde?

Reply:
Using v12 everywhere else and I think spec-generator is on v12 also. So, should move all of them together. Breaking change?
Node v12 goes into maintenance 2020-11-30 FWIW.

Reply:
We should be tied to LTS instead (everywhere)... 12 seems way too limiting.

Reply:
Started https://github.com/w3c/respec/pull/3450